dolphinscheduler-ui/src/js/module/i18n/locale/en_US.js

```diff
@@ -187,7 +187,6 @@ export default {
   'Please select a queue': 'default is tenant association queue',
   'Please enter the tenant code in English': 'Please enter the tenant code in English',
   'Please enter tenant code in English': 'Please enter tenant code in English',
-  'Please enter tenant name without special characters': 'Please enter tenant name without special characters',
```

Comment:
This file modification should have nothing to do with this PR.

Reply:
Corrected back！